internal/pkg/cli/pipeline_delete_test.go

```diff
@@ -238,19 +185,41 @@ func TestDeletePipelineOpts_Ask(t *testing.T) {
 				require.NoError(t, err)
 				require.Equal(t, tc.wantedAppName, opts.appName, "expected app names to match")
 				require.Equal(t, tc.wantedPipelineName, opts.name, "expected pipeline names to match")
-				require.Equal(t, tc.wantedSecret, opts.ghAccessTokenSecretName, "expected secrets to match")
 			}
 		})
 	}
 }
 
 func TestDeletePipelineOpts_Execute(t *testing.T) {
+	mockResp := &sdkSecretsmanager.DescribeSecretOutput{
+		ARN:         aws.String("arn-goose"),
+		CreatedDate: aws.Time(time.Now()),
```

Comment:
i wonder if we should create a `now := time.Now()` at the top of the function to get reused. i don't think there's any guarantee that the struct will get created at exact the same time (although in some light testing, it usually is) so i'm worried the test may fail randomly

Reply:
That's a great idea!